tfhub_dev/assets/intel/midas/v2_1/2/lite/1.md

```diff
@@ -0,0 +1,90 @@
+# Lite intel/midas/v2_1/2
```

Comment:
This should follow the direcotry structure, so it should be `intel/midas/v2_1/2/lite/1`